frontend/src/helpers/editActionDialogHelper.js

```diff
@@ -0,0 +1,90 @@
+function isEmailFormEmpty(emailType, emailTo, emailCc, emailResponse, reasonsForActionContent) {
```

Comment:
Need lots of comments and tests here.

Reply:
Added comments in commit #https://github.com/code-for-canada/project-thundercat/pull/229/commits/2ffba3713cadbb590f4fcc512a45eaaa15c24c65

Tests are coming...

Reply:
You did add comments to EditActionDialog - making it very navigable. However, it would be good to have some in here too, at least at the tops of the functions. Most of the logic is straight forward, but a couple lines about the function and the inputs would be ideal.

Reply:
Added in commit #https://github.com/code-for-canada/project-thundercat/pull/229/commits/86a746a0445de26b3c0742fd3f8de0c1f7d5affb